frontend/server/src/UITools.php

```diff
@@ -207,6 +207,14 @@ private static function getCommonPayload(
                 \OmegaUp\Controllers\User::getUserTypes($user, $identity) :
                 []
             ),
+            'nextRegisteredContestForUser' => (
+                !is_null($identity) &&
+                !is_null($identity->identity_id) ?
+                \OmegaUp\Controllers\Contest::getNextRegisteredContestForUser(
+                    $identity
+                ) :
+                []
+            ),
```

Comment:
Entiendo que en los demás parámetros se realizan algunas validaciones antes de mandar llamar la función que obtendrá el valor deseado, pero ¿que te parece si mejor aquí sólo agregamos el llamado a la función del DAO yesa validación la hacemos en el mismo DAO?
```suggestion
            'nextRegisteredContestForUser' => \OmegaUp\DAO\Contests::getNextRegisteredContestForUser(
                    $identity
            ),
```
La validación podría quedar de la siguiente manera:
```php
        if (is_null($identity) || is_null($identity->identity_id)) {
            return null;
        }
```
y esa se agregaría al inicio de la función `getNextRegisteredContestForUser`